File: app/src/main/java/com/theappsolutions/boilerplate/util/other/LogUtils.java
package com.theappsolutions.boilerplate.util.other;

import timber.log.Timber;

/**
 * @author Dmytro Yakovlev d.yakovlev@theappsolutions.com
 * @copyright (c) 2017 TheAppSolutions. (https://theappsolutions.com)
 */
public class LogUtils {

    public static void logThread(String str) {
        String name = Thread.currentThread().getName();
        str += " [ " + name + " ] ";
        Timber.d(str);
    }
}

File: app/src/main/java/com/theappsolutions/boilerplate/util/other/LogUtils.kt
package com.theappsolutions.boilerplate.util.other

import timber.log.Timber

/**
 * @author Dmytro Yakovlev d.yakovlev@theappsolutions.com
 * @copyright (c) 2017 TheAppSolutions. (https://theappsolutions.com)
 */
    fun logThread(str: String) {
        var strTread = str
        val name = Thread.currentThread().name
        strTread += " [ $name ] "
        Timber.d(str)
    }
